### PR TITLE
add labels to open rosa auth

### LINKED ISSUE #422: Add section labels to openrosa pages
I don't know why I didn't before, but I need to do so.

### PR DESCRIPTION
**Pull request by adammichaelwood**
closes #422

#### What is included in this PR?

adds labels to openrosa-authentication

(contra the issue description, most of the OR pages had labels. Just that one didn't.)

### CODE REVIEW

File: odk1-src/openrosa-authentication.rst [hunk 1]
```diff
@@ -98,14 +108,16 @@ Clients MUST NOT include authentication credentials in the URL to the server. Th
   
   http://username:password@myhost.org/mypage
 
+.. _openrosa-authentication-basic:
```
Comment (yanokwa) [inline odk1-src/openrosa-authentication.rst]: I'd add s/basic/basic-authentication for consistency

File: odk1-src/openrosa-authentication.rst [hunk 2]
```diff
@@ -98,14 +108,16 @@ Clients MUST NOT include authentication credentials in the URL to the server. Th
...
+  
 Basic Authentication
 ~~~~~~~~~~~~~~~~~~~~~~~~
     
 Basic Authentication MUST NOT be performed over a non-secure (HTTP) connection.
 
 Once the client is aware that basic authentication is required, it SHOULD proactively supply the basic authentication credentials on every secure request to the server, rather than wait for the server to reject the request with a ``401`` response.
 
-.. _openrosa-restrcted-digest-authentication:
+.. _openrosa-restricted-digest-authentication:
```
Comment (yanokwa) [inline odk1-src/openrosa-authentication.rst]: Are labels not spell checked?

File: odk1-src/openrosa-authentication.rst [hunk 3]
```diff
@@ -28,6 +30,8 @@ OpenRosa compliant servers MUST support at least one of either:
 
 We are following RFC2617 with additional OpenRosa compliance requirements defined in the implementation section below to ensure that the Digest Authentication implementations across devices do not compromise security and that they all implement a well-defined common subset of the RFC2617 Digest Authentication mechanism.
 
+.. _openrosa-authentication-security:
```
Comment (yanokwa) [inline odk1-src/openrosa-authentication.rst]: Why not s/security/data-security

File: odk1-src/openrosa-authentication.rst (hunk 2 above)
Comment (adammichaelwood) [reply odk1-src/openrosa-authentication.rst]: No they aren't.

**Pull request merged**